modules/rosa-deleting-cluster.adoc

```diff
@@ -22,6 +29,13 @@ If add-ons are installed, the deletion takes longer because add-ons are uninstal
 ----
 $ rosa delete cluster --cluster=<cluster_name> --watch
```

Comment:
Can we make `--watch` optional?

Reply:
@0kashi thank you for your review. 

I tested this both with and without `--watch`. If you don't specify that option, you have to run the `rosa logs uninstall -c pneedlerosa --watch` command anyway, to know when the deletion is complete. Since the subsequent steps to delete the IAM resources must not be carried out until the cluster is deleted, my view is that this isn't really optional. I.e. to complete the deletion procedure, you must either run `--watch` when you delete the cluster, or run the second command.

My view is that for this procedure, we should pick one and not state that it's optional. I.e. we should either leave `--watch` out and add the additional command, or leave this as part of the recommended command.

Reply:
I tested this again this morning. Without the `--watch` argument, the command returns the following ouput and then exits, leaving the cluster to uninstall without a success or failure notification being provided:

```
$ rosa delete cluster --cluster=pneedlerosa
? Are you sure you want to delete cluster pneedlerosa? Yes
I: Cluster 'pneedlerosa' will start uninstalling now
I: Your cluster 'pneedlerosa' will be deleted but the following object may remain
I: Operator IAM Roles: - arn:aws:iam::269733383066:role/pneedlerosa-n9g1-openshift-cloud-credential-operator-cloud-crede
 - arn:aws:iam::269733383066:role/pneedlerosa-n9g1-openshift-image-registry-installer-cloud-creden
 - arn:aws:iam::269733383066:role/pneedlerosa-n9g1-openshift-ingress-operator-cloud-credentials
 - arn:aws:iam::269733383066:role/pneedlerosa-n9g1-openshift-cluster-csi-drivers-ebs-cloud-credent
 - arn:aws:iam::269733383066:role/pneedlerosa-n9g1-openshift-machine-api-aws-cloud-credentials

I: OIDC Provider : https://rh-oidc.s3.us-east-1.amazonaws.com/1p2cs4vq6leorch3mdeaa4oahla5lp0e

I: Once the cluster is uninstalled use the following commands to remove the above aws resource.

	rosa delete operator-roles -c 1p2cs4vq6leorch3mdeaa4oahla5lp0e
	rosa delete oidc-provider -c 1p2cs4vq6leorch3mdeaa4oahla5lp0e
I: To watch your cluster uninstallation logs, run 'rosa logs uninstall -c pneedlerosa --watch'
```

I think we should do one of the following things:

- Leave the `--watch` argument as in a recommended part of the `rosa delete cluster` step
- Remove `--watch` and add the `rosa logs uninstall ...` command as an additional step
- Update the `rosa delete cluster` command so that it runs in the foreground until the deletion has been completed, then return a success or failure notification.